src/engine/sparqlExpressions/NaryExpression.h

```diff
@@ -82,6 +82,7 @@ SparqlExpression::Ptr makeIfExpression(SparqlExpression::Ptr child1,
                                        SparqlExpression::Ptr child2,
                                        SparqlExpression::Ptr child3);
 
+// Implemented in ConvertToNumeric.cpp
 SparqlExpression::Ptr makeIntExpression(SparqlExpression::Ptr child);
```

Comment:
As they are not used very often, maybe use the more explicit name `makeConvertToIntExpression` (same for the double) as it is more consistent with the naming of the file etc.